src/views/dashboards/Dashboard/index.vue

```diff
@@ -33,20 +40,56 @@
         </button>
       </section>
     </header>
+    <section class="dashboard__widgets" v-if="firstSection">
+      <div class="dashboard__widgets-cards">
+        <article
+          class="dashboard__widgets-card"
+          v-for="(card, index) in cards"
+          :key="index"
+        >
+          <h4 class="dashboard__widgets-card-title">{{ card.title }}</h4>
+          <p class="dashboard__widgets-card-description">
+            {{ card.description }}
+          </p>
+        </article>
+      </div>
+      <div>
+        <ColumnCharts :chartData="chartData" />
+        <AgentChats />
+      </div>
```

Comment:
```suggestion
      <section class="dashboard__widgets-cards">
        <article
          class="dashboard__widgets-card"
          v-for="(card, index) in cards"
          :key="index"
        >
          <h4 class="dashboard__widgets-card-title">{{ card.title }}</h4>
          <p class="dashboard__widgets-card-description">
            {{ card.description }}
          </p>
        </article>
      </section>
      <section>
        <ColumnCharts :chartData="chartData" />
        <AgentChats />
      </section>
```

Reply:
It would be good to add a class in this second section to better describe its context/function